<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QWidget" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="colorIdLabel">
       <property name="text">
        <string>Color ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="colorSourceText">
       <property name="plainText">
        <string>Color source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="colorDestText">
       <property name="plainText">
        <string>Color Dest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
